package.json

```diff
@@ -22,6 +22,9 @@
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -p ./",
 		"watch": "tsc -watch -p ./",
+		"postinstall": "node ./node_modules/vscode/bin/install",
+		"test": "npm run compile && node ./node_modules/vscode/bin/test",
```

Comment:
Those two lines ("postinstall" and "test") are no longer needed. Could you remove them?

Reply:
Well, it sounds like it goes beyond the scope of my PR :)
Feel free to commit those changes directly but you probably should do another PR or do it directly in master as this is absolutely not related to the issue the PR is fixing.

Reply:
They were actually removed in #3, the PR I merged after you create this PR. However those lines are going to be reintroduced by this PR. It's probably accidentaly included when resolving merge conflicts. Sorry for confusion.

Reply:
Merging #2 before #3 would have been a good idea 😄
Done